unified-codes/apps/data-service/src/app/v1/types/DgraphDataSource.ts

```diff
@@ -178,15 +178,19 @@ export class DgraphDataSource extends RESTDataSource {
         }
         parents: ~children {
           type: dgraph.type
-          description: name@*
+          description: name
```

Comment:
Is this an important change? Any idea what it had @*?

Reply:
That's the other component of removing the `@lang` directive... `@*` was querying a name in any language